edit suggestions (#39)
w3c/wai-intro-atag: content/no-code.md

## Why Accessibility Matters

> Your no-code tool users include people with disabilities. Your tool needs to be accessible to disabled content creators.

Reviewer: I am good with this change. "disabled content creators" is a bit different from our traditional "... with disabilities". I am happy for us to get closer to the social model of disability, but just wanted to raise this in case we do want to remain consistent with people-first language.

Author: Correct. We are using both "people with disabilities" and "disabled people" now. We need to update our documentation on that. (sidenote: we are also using both in the videos.)

> Your no-code tool users include people with disabilities. Your tool needs to be accessible to disabled content creators.

Reviewer: ```suggestion
People with disabilities need to use your no-code tool. Your tool needs to be accessible to disabled content creators.
```

> The websites that your tool outputs also need to be accessible to disabled website users.

Reviewer: ```suggestion
The websites and widgets that your tool outputs also need to be accessible to disabled website users.
```

These tools may also output components or widgets. These may or may not be part of the whole website. Is it worth for uss to make that difference?

Author: I don't know. Perhaps leave it simply just websites for now and leave an open issue to revisit when we can get more input. Maybe open a new GitHub issue and ask EOWG to comment?

Reviewer: Thanks. I'll add for now and open separate issue in case we decide to go back.

### Everyone can add blocks of content

<p><span class="persona">Persona: </span>
 Mohamed is an independent consultant who has a website to attract clients. He is blind and uses a screen reader that reads aloud the information on his screen.
</p>
<div class="quotes">
  <ul>
    <li>
      <p><span class="issue">Problem: </span><span><q>
        I want to add a new section to my home page. The tool documentation says to drag and drop a content block. I can't do that, because I can't use a mouse. I tried using my screen reader’s built-in functionality to drag and drop, but it doesn't work with this tool.
      </q></span></p>
    </li>
    <li>
      <p><span class="issue">Works well: </span><span><q>
        I can use the keyboard to select the content block and put it where I want on the page. Everything is announced properly, including the block type and the location where I place it.
      </q></span></p>
    </li>
  </ul>
</div>

> Users who have auditory, cognitive, neurological, physical, speech, and visual disabilities need to be able to add, customize, and remove blocks of content.

Reviewer: Why are you proposing to remove the second sentence here?

### The tool provides accessible components and templates

<p><span class="persona">Persona: </span>
  Amahle is a hairdresser and is proud that she created her own website. Clients use her website to schedule appointments, and it's a real time saver for her.
</p>

<div class="quotes">
  <ul>
    <li>
      <p><span class="issue">Problem: </span><span><q>
        Some of my older clients say they cannot use the calendar widget to schedule an appointment. One says she uses large text and the buttons are cut off. Others have complained that the light gray text is too hard to read.
      </q></span></p>
    </li>
    <li>
      <p><span class="issue">Works well: </span><span><q>

>         The calendar widget that the tool provides works great for all my clients, including the older clients. The tool helped me pick color combinations that are easy to see.

Reviewer: ```suggestion
        The calendar widget that the tool provides works great for all my clients, including the older clients. The tool helped me pick color combinations that are easy to see, and guides me to add headings, lists, and labels.
```

Not sure why this second part was removed. I think it fits well here and ties well to the paragraph below.

Reviewer: I think we should list them in both places. Agree that does not closely relate to the persona, but it makes for an accessible output, and I think it's relevant to have hear.
Will also open a separte issue for discussion.

<p><span class="persona">Persona: </span>
  Dacso runs an automotive repair shop and he does a lot of body work. His daughter created a website for him to showcase his work. Dacso uploads before-and-after photos of his projects. He also posts videos with simple maintenance tips.
</p>
<div class="quotes">
  <ul>
    <li>
      <p><span class="issue">Problem: </span><span><q>
        I usually get lots of great feedback on my website. A couple of Deaf people say they want to learn from my videos, but of course they can't hear what I'm saying. I really want to make it so they can get advice from my videos, but I have no idea what to do.
      </q></span></p>
    </li>
    <li>
      <p><span class="issue">Works well: </span><span><q>
        Whenever I upload a picture, the tool asks me for a text description. It has "more info" that explained how to write a good text description for people who are blind and can't see the image. It also has good support for adding captions to videos. And the "more info" for videos explained how to describe what I'm doing when I make the video, for people who don't see too well.
      </q></span></p>
    </li>
  </ul>
</div>

> Your tool should enable and encourage content creators to provide alternative text for images and for videos. It should include guidance on how to provide useful alternatives for their website consumers who have disabilities.

Reviewer: ```suggestion
Your tool should enable and encourage content creators to provide alternative text for images, audio, and videos. It should include guidance on how to provide useful alternatives for their website consumers who have disabilities.
```

I think otherwise it could be read as if audio-only does not need alternative.

**To get started putting ATAG to work for you, see**:

* **[[ATAG Overview]](/standards-guidelines/atag/)**, with links to the ATAG standard and Implementing ATAG

> * **[[ATAG at a Glance]](/standards-guidelines/atag/glance/)**, a paraphrased summary to give you an idea of what's covered in ATAG

Reviewer: Not sure what's changed here.

Author: blank line at end of file
